cmd/virt-handler/Dockerfile

```diff
@@ -20,9 +20,18 @@ FROM fedora:26
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
-RUN dnf -y install libvirt-client \
+RUN dnf -y install libvirt-client cyrus-sasl-scram \
     && dnf -y clean all
 
 COPY virt-handler /virt-handler
+COPY virt-handler.sh /virt-handler.sh
+RUN chmod a+x /virt-handler.sh
```

Comment:
Combine these two RUN lines.  Less layers in the union mount filesystem == better performance.